src/etl.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
+
+from utils import data
+
+# Convert timestamp to hourly level
+def convert_timestamp_to_hourly(df: pd.DataFrame = None, column: str = None) -> pd.DataFrame:
+    """
+    Convert timestamp to hourly level
+
+    Args:
+        df (pd.DataFrame, optional): Input dataframe. Defaults to None.
+        column (str, optional): Column related to datetime data. Defaults to None.
+
+    Returns:
+        DataFrame: resultant dataframe with hourly timestamps.
+    """
+    dummy = df.copy()
+    dummy[column] = pd.to_datetime(dummy[column], format='%Y-%m-%d %H:%M:%S')  # String to datetime datatype conversion
+    dummy[column] = dummy[column].dt.floor('h')  # Truncate timestamps to beginning of hour
+    return dummy
+    
+
+# Aggregate data
+sales_df = convert_timestamp_to_hourly(data('data/sales.csv'),'timestamp')
+salesagg_df = sales_df.groupby(by=['timestamp','product_id']).agg({'quantity' : 'sum'}).reset_index()
+salesagg_df.to_csv('data/sales_agg.csv',index=False)
+
+stocklevel_df = convert_timestamp_to_hourly(pd.read_csv('data/sensor_stock_levels.csv'),'timestamp')
```

Comment:
You haven't replace read_csv everywhere

Reply:
best to do relevant changes in the same PR